lib/slack.ex

```diff
@@ -171,6 +171,20 @@ defmodule Slack do
   end
 
   @doc """
+  Sends `text` and `attachments` to `channel` for the given `slack` connection.
+  """
+  def send_attachments(text, attachments, channel, slack) do
```

Comment:
I've added the `is_list` guard. I'm pretty new to elixir's way of doing things, should I define a catch-all case and raise manually or should I just allow the FunctionClauseError when someone attempts to use this with something other than a list of `attachments`?